tutor/resources/styles/book-content/base.scss

```diff
@@ -63,7 +63,6 @@
   section:not(.references) h1 + p:first-of-type,
   [data-type="document-title"] ~ p:first-of-type {
     margin-top: 2rem;
-    @include tutor-reading-first-letter();
   }
```

Comment:
Awesome!  can you also do a search for `tutor-reading-first-letter` and remove the mixin if it's not in use anywhere else? I suspect this was the only spot it was referenced.

Reply:
Good call, this was unreferenced and unlikely to be used in the future. Removed in a follow up commit 👍